pkg/server/datastore/sqlstore/sqlstore.go

```diff
@@ -3818,6 +3818,9 @@ func pruneRegistrationEntries(tx *gorm.DB, expiresBefore time.Time, logger logru
 		if err := deleteRegistrationEntrySupport(tx, entry); err != nil {
 			return err
 		}
+		if err := createRegistrationEntryEvent(tx, entry.EntryID); err != nil {
```

Comment:
Can we add coverage for actually pruning events? It looks like this test case prunes zero and expects zero entries: https://github.com/spiffe/spire/blob/main/pkg/server/datastore/sqlstore/sqlstore_test.go#L4028-L4032

Reply:
updated existing unit test to cover this case. lmk if it looks ok to you.

Reply:
It's very similar to @stevend-uber's adjustment to this unit test. It covers event creation so, yes, looks OK. Thanks for updating.

Reply:
Thats good. Thanks for confirming